common/src/main/java/com/linecorp/centraldogma/common/RevisionRange.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.common;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * A class which contains {@link Revision}s of {@code from} and {@code to}.
+ */
+public class RevisionRange {
+
+    private final Revision from;
+
+    private final Revision to;
+
+    /**
+     * Creates a new instance with the specified integer value of {@code from} and {@code to}.
+     */
+    public RevisionRange(int from, int to) {
+        this(new Revision(from), new Revision(to));
+    }
+
+    /**
+     * Creates a new instance with the specified {@code from} {@link Revision} and {@code to} {@link Revision}.
+     */
+    public RevisionRange(Revision from, Revision to) {
+        this.from = requireNonNull(from, "from");
+        this.to = requireNonNull(to, "to");
+    }
+
+    /**
+     * Returns the {@code from} {@link Revision}.
+     */
+    public Revision from() {
+        return from;
+    }
+
+    /**
+     * Returns the {@code to} {@link Revision}.
+     */
+    public Revision to() {
+        return to;
+    }
+
+    /**
+     * Returns the {@link RevisionRange} whose major value of {@code from} {@link Revision} is lower than
+     * or equal to the major value of {@code to} {@link Revision}.
+     *
+     * @throws IllegalStateException if the {@code from} and {@code to} {@link Revision}s are in the
+     *                               different state. they should be either absolute or relative
```

Comment:
nit: I'm not sure..  `they`->`They` and adding a dot(`.`) to the end of the sentence?

Reply:
Thank you for approving this.
This is one of the hardest stuff that I cannot make a decision.
We do not start with an uppercase and do not end with dot when there is just only one sentence.
But how can I do when there are two sentences? Do I need to use uppercase and dot? Do we have a convention for this? :)